src/anbox/cmds/session_manager.cpp

```diff
@@ -194,12 +195,11 @@ anbox::cmds::SessionManager::SessionManager()
     const auto should_force_software_rendering = utils::get_env_value("ANBOX_FORCE_SOFTWARE_RENDERING", "false");
     auto gl_driver = graphics::GLRendererServer::Config::Driver::Host;
     if (should_force_software_rendering == "true" || use_software_rendering_)
-     gl_driver = graphics::GLRendererServer::Config::Driver::Software;
+      gl_driver = graphics::GLRendererServer::Config::Driver::Software;
 
-    graphics::GLRendererServer::Config renderer_config {
-      gl_driver,
-      single_window_
-    };
+    graphics::GLRendererServer::Config renderer_config{
+        gl_driver,
+        single_window_};
```

Comment:
As you can see there are quite a few formatting changes (done using clang-format), are we ok with them?

Reply:
Yeah that is fine.